pay/test/Pay.Upfront.test.ts

```diff
@@ -161,9 +174,9 @@ describe('HectorUpfrontPay', function () {
         upgradeableAdmin.address
       );
     });
-    it('pay subscription', async function () {
-      expect(await hectorPayFactory.subscription()).equal(
-        hectorSubscription.address
+    it('pay validator', async function () {
```

Comment:
@bejaxer need another test case to test isValid when passing a valid and failed data